core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -496,40 +490,52 @@ object KafkaConfig {
   val S3SpanMaxQueueSizeProp = "s3.telemetry.tracer.span.max.queue.size"
   val S3SpanMaxBatchSizeProp = "s3.telemetry.tracer.span.max.batch.size"
 
-  val S3EndpointDoc = "The S3 endpoint, ex. <code>https://s3.{region}.amazonaws.com</code>."
-  val S3RegionDoc = "The S3 region, ex. <code>us-east-1</code>."
-  val S3PathStyleDoc = "The S3 path style, ex. <code>true</code>."
-  val S3BucketDoc = "The S3 bucket, ex. <code>my-bucket</code>."
-  val S3WALPathDoc = "The S3 WAL path. It could be a block device like /dev/xxx or file path in file system"
-  val S3WALCapacityDoc = "The S3 WAL capacity. The value should be larger than s3.wal.cache.size cause of log storage format may not compact."
+
+  val ElasticStreamEnableDoc = "Whether to enable AutoMQ, it has to be set to true"
+  val ElasticStreamEndpointDoc = "Specifies the Elastic Stream endpoint, ex. <code>es://hostname1:port1,hostname2:port2,hostname3:port3</code>.\n" +
+    "You could also PoC launch it in memory mode with endpoint <code>memory:://</code> or redis mode with <code>redis://.</code>"
+  val ElasticStreamNamespaceDoc = "The kafka cluster in which elastic stream namespace which should conflict with other kafka cluster sharing the same elastic stream."
+  val S3EndpointDoc = "The object storage endpoint, ex. <code>https://s3.us-east-1.amazonaws.com</code>."
+  val S3RegionDoc = "The object storage region, ex. <code>us-east-1</code>."
+  val S3PathStyleDoc = "The object storage access path style. When using MinIO, it should be set to true."
+  val S3BucketDoc = "The object storage bucket."
+  val S3WALPathDoc = "The local WAL path for AutoMQ can be set to a block device path such as /dev/xxx or a filesystem file path." +
+    "It is recommended to use a block device for better write performance."
+  val S3WALCapacityDoc = "The size of the local WAL for AutoMQ. This determines the maximum amount of data that can be written to the buffer before data is uploaded to object storage." +
+    "A larger capacity can tolerate more write jitter in object storage. The value should be larger than s3.wal.cache.size"
```

Comment:
Now, there is no strict size relationship between `s3.wal.capacity` and `s3.wal.cache.size`.